File: client/butterfly/app/src/main/java/pf/com/butterfly/message/Protocols/bored_head_item_add_req.java
package pf.com.butterfly.message.Protocols;
import android.util.Log;
import org.json.JSONArray;
import org.json.JSONObject;
import pf.com.butterfly.message.MsgBase;
import pf.com.butterfly.message.MsgCodeId;
  public class bored_head_item_add_req extends MsgBase
  {
      public bored_head_item_add_req()
      {
          CodeId = MsgCodeId.bored_head_item_add_req;
      }
      public String content;    //

}

File: server/hudie/hudie/message/Protocols/wu_liao/bored_head_item_add_req.cs
using messages;

namespace messages.Protocols
{
  public class bored_head_item_add_req : MsgBase
  {
      public bored_head_item_add_req()
      {
          CodeId = MsgCodeId.bored_head_item_add_req;
      }
      public string content;    //

	}
}
